src/app/country/show-city/show-city.component.ts

```diff
@@ -0,0 +1,11 @@
+import { Component, Input } from '@angular/core';
+import { Country } from '../../models/country';
+
+@Component({
+  selector: 'app-show-city',
+  templateUrl: './show-city.component.html',
+  styleUrls: ['./show-city.component.css'],
+})
+export class ShowCityComponent {
```

Comment:
Verbs like "show" are used in methods. You should not use it in Components. CityComponent or CapitalCityComponent is fine.